mediamonks/rest-api-bundle/3.0/config/packages/mediamonks_rest_api.yaml

```diff
@@ -0,0 +1,20 @@
+mediamonks_rest_api:
+    request_matcher:
+        path: /api
+
+    # manually force debug to be enabled or disabled
+    #debug: %kernel.debug%
+
+    # set the post message origin when needed
+    #post_message_origin:
+
+    # use regular expressions instead of a path to enable this bundle
+    #request_matcher:
```

Comment:
this gets even worse here. Uncommenting these lines would make the Yaml invalid due to duplicate keys.